src/Forms/GridField/GridField_ActionMenu.php

```diff
@@ -31,15 +31,20 @@ public function getColumnContent($gridField, $record, $columnName)
             return null;
         }
 
-        $schema = array_map(function (GridField_ActionMenuItem $item) use ($gridField, $record, $columnName) {
-            return [
+        $schema = [];
+        /* @var GridField_ActionMenuItem $item */
+        foreach ($items as $item) {
+            if (!$item->getGroup($gridField, $record, $columnName)) {
```

Comment:
You're calling `getGroup()` twice. Looks like it would be better to assign the return value to a variable and work from that.